ci-operator/step-registry/openshift/upgrade/aws/single-node/openshift-upgrade-aws-single-node-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+workflow:
+  as: openshift-upgrade-aws-single-node
+  steps:
+    allow_best_effort_post_steps: true
+    pre:
+    - ref: ipi-install-hosted-loki
+    - chain: ipi-conf-aws
+    - ref: single-node-conf
+    - chain: ipi-install-stableinitial
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - ref: openshift-e2e-test-capabilities-check
+    - chain: gather-network
+    - chain: gather-core-dump
+    - chain: ipi-aws-post
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: "release:initial"
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: "release:latest"
+    env:
```

Comment:
We should add `TEST_SKIPS` here - most of the disruption tests are invalid for SNO

Reply:
I see you already added it in yours. Updating this PR to only focus on serial